client/signup/steps/design-type-with-store/new-site-image.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default () => (
+	<svg width="172px" height="112px" viewBox="0 0 172 112" version="1.1" xmlns="http://www.w3.org/2000/svg">
+		<defs>
+			<polygon id="path-1" points="0 15.7603469 171.708196 15.7603469 171.708196 111.769139 0 111.769139"></polygon>
+			<mask id="mask-2" maskContentUnits="userSpaceOnUse" maskUnits="objectBoundingBox" x="0" y="0" width="171.708196" height="96.0087922" fill="white">
+				<use xlinkHref="#path-1"></use>
+			</mask>
+		</defs>
+		<g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+			<g id="Group-3-Copy">
+				<g id="Group-2">
+					<g id="domain-illustration">
+						<g id="Imported-Layers">
+							<path d="M0.102747909,15.6269981 L0.102747909,2.72646653 C0.102747909,1.30182267 1.26010036,0.147016357 2.6878853,0.147016357 L169.225395,0.147016357 C170.653386,0.147016357 171.810944,1.30223275 171.810944,2.72708166 L171.810944,15.6269981 L0.102747909,15.6269981" id="Fill-1" fill="#CFDEE7"></path>
+							<use id="Fill-6" stroke="#E6F0F6" mask="url(#mask-2)" stroke-width="2" fill="#FFFFFF" xlinkHref="#path-1"></use>
```

Comment:
`stroke` should be `#D9E8F0` to darken the illustration stroke and match the domain-only illustration.